ytmusicapi/mixins/playlists.py

```diff
@@ -270,3 +274,17 @@ def remove_playlist_items(self, playlistId: str, videos: List[Dict]) -> Union[st
         endpoint = 'browse/edit_playlist'
         response = self._send_request(endpoint, body)
         return response['status'] if 'status' in response else response
+
```

Comment:
Please don't create a utility function for it. Two reasons:

- the functionality is not used anywhere else
- it would be exposed to the library caller, which is not needed

Instead just put the code right into `add_playlist_items`.